doc/manual/developing.rst

```diff
@@ -4,7 +4,7 @@ Developing ARTIQ
 ^^^^^^^^^^^^^^^^
 
 .. warning::
-    This section is only for software or FPGA developers who want to modify ARTIQ. The steps described here are not required if you simply want to run experiments with ARTIQ. If you purchased a system from M-Labs or QUARTIQ, we normally provide board binaries for you.
+    This section is only for software or FPGA developers who want to modify ARTIQ. If you want to modify firmware for Kasli-SoC or other ARM boards, please refer to `artiq-zynq repo <https://git.m-labs.hk/M-Labs/artiq-zynq>`_. The steps described here are not required if you simply want to run experiments with ARTIQ. If you purchased a system from M-Labs or QUARTIQ, we normally provide board binaries for you.
```

Comment:
1. *the* artiq-zynq repository, or, imo, 'ARTIQ-Zynq' / 'ARTIQ on Zynq' 
2. I'm fine with adding this now, because any information is better than no information, but it's worth mentioning that linking to artiq-zynq doesn't help a great deal here because the documentation there is just as bad. I have ambitions of making this manual page complete and useful (it's been complained about enough in issues, forum, etc.) which I've been collecting information for and will with any luck get around to, so the situation hopefully will improve at some point soon

Reply:
1. Edited
2. I would just add that I'm adding it after we had an incident at helpdesk with it.

Reply:
1. Sorry, I should have phrased more clearly -- *the* is necessary regardless, grammatically 
2. Yes, not a criticism of adding the line now! More of an FYI